### PR TITLE
Update hardware.md

### PR DESCRIPTION
**Pull request by tauanbinato**
Added the nuc freeze NOTE session for users that are about to invest. 

I added all the information i could gather since i have this issue myself. In discord  #hardware has a topic for that too.

### CODE REVIEW

**Review by tauanbinato — COMMENTED** [review]
I just notice i forgot the link on line 254
[guide]().

here if the revisor can add:  [guide](https://docs.oracle.com/cd/E19407-01/820-7898-11/z40001c91006831.html) 

Thanks

**Review by mrnadyy — APPROVED** [review]

File: docs/guides/node/local/hardware.md [hunk 1]
```diff
@@ -231,6 +231,33 @@ NUCs are great for most stakers that only run a few validators because of their
 Unlike PCs, NUCs come preassembled in a case; all you need to do is add some RAM, add an SSD, and you're up and running!
 Below are a few examples of NUC setups that some Rocket Pool veterans use and recommend.
 
+::: tip NOTE
+**Ethernet Adaptor Compatibility**
+
+If you're planning to buy an Intel® NUC 11th or 12th Generation you can encounter connectivity issues with the ethernet adaptor, specifically if the adaptor is identified as **I225-LM** (Check intel especifications before buying).
```
Comment (tauanbinato) [inline docs/guides/node/local/hardware.md:237]: specifications no especifications

File: docs/guides/node/local/hardware.md [hunk 2]
```diff
@@ -231,6 +231,33 @@ NUCs are great for most stakers that only run a few validators because of their
...
+
+**Driver and Software Updates**: Consider updating your drivers, firmware, and BIOS by referring to the official Intel® support page for your NUC model [here](https://www.intel.com/content/www/us/en/search.html?ws=text#sort=relevancy&f:@tabfilter=[Downloads). This might include using the latest available support driver from Intel's website or applying BIOS updates that address compatibility concerns.
+
+**Intel's Patch (Windows)**: Intel has released a patch to address a similar issue on Windows systems. While the patch itself **may not directly apply to Linux environments**, it highlights the recognition of the problem by Intel and their efforts to provide solutions. You can find more details about the patch in this [link](https://www.intel.com/content/www/us/en/download/705968/patch-for-a-modern-standby-lan-issue-on-intel-nuc-11th-12th-generation-products.html?wapkw=nuc11tnhi3).
+For further assistance on driver installation and software updates in Linux, you can refer to this [guide]().
```
Comment (tauanbinato) [inline docs/guides/node/local/hardware.md:254]: [guide](https://docs.oracle.com/cd/E19407-01/820-7898-11/z40001c91006831.html)

**Review by tauanbinato — COMMENTED** [review]
Some details on lines

**Pull request merged**